src/apps/util/db.clj

```diff
@@ -0,0 +1,57 @@
+(ns apps.util.db
+  "This namespace contains some database utility functions. For the time being, these functions will use Korma to
+   obtain database connections. This code will have to be modified when Korma is removed."
+  (:require [apps.util.conversions :refer [date->timestamp]]
+            [korma.core :as sql]
+            [korma.db :as db]))
+
+(defmacro transaction
+  "Executes queries within the body in a single transaction. Nested transactions are absorbed into the outermost
+   transaction. For now, this is just an alias to korma.db/transaction. When Korma is removed, it will be necessary
+   to come up with an alternative implementation."
```

Comment:
This is of course out of scope for this PR, but it might be nice to eventually do this with https://github.com/seancorfield/next-jdbc once Korma is out of the picture.